lib/vrl/compiler/src/expression.rs

```diff
@@ -331,29 +333,25 @@ impl DiagnosticError for ExpressionError {
     }
 
     fn message(&self) -> String {
-        use ExpressionError::*;
```

Comment:
Any reason to remove this? Is it a personal preference, or an actual requirement?

I don't mind if it's a personal preference. I added it because my preference is the opposite, but I'm fine keeping your change in.

Reply:
Ah yes I think this was a hold-over from when I introduced an `Abort` struct that I backed out such that it was complaining about shadowing. I prefer this style too so I'll re-instate.